composables/useUserAddresses.ts

```diff
@@ -0,0 +1,39 @@
+import { computed } from "@vue/composition-api"
```

Comment:
I think we need to import from @nuxtjs/composition-api

Reply:
Resolved.